### ISSUE TITLE
duplicate "package object scalapb"

### ISSUE THREAD

**Issue by xuwei-k** [scalapb/ScalaPB#777]
compiler-plugin and scalapb-runtime have `package object scalapb`. can't use together.

- https://github.com/scalapb/ScalaPB/blob/03a7c8d63326ec18410c09a66d778c8b7766a958/compiler-plugin/src/main/scala/scalapb/package.scala
- https://github.com/scalapb/ScalaPB/blob/03a7c8d63326ec18410c09a66d778c8b7766a958/scalapb-runtime/shared/src/main/scala/scalapb/package.scala

**Comment by thesamet** [comment]
Thanks for reporting. Is there an actual usage in existence for those two packages together that we need to solve for? Should be pretty uncommon.

**Comment by thesamet** [comment]
Re-opening as the fix introduces an incompatibility with akka-grpc. This will be resolved in 0.11.0 where I can remove the scalapb package object from runtime.

**Comment by xuwei-k** [comment]
Could not use scalapb with sbt 1.4.0-M1 due to this issue 😢 

```
[error] java.lang.NoSuchMethodError: scalapb.package$.gen$default$1()Z
[error] Use 'last' for the full log.
[warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
```

sbt 1.4.0-M1 depends on zinc 1.4.0-M6, zinc 1.4.0-M6 depends on scalapb-runtime 0.10.2.

- https://github.com/sbt/zinc/blob/v1.4.0-M6/project/plugins.sbt#L9
- https://github.com/sbt/sbt/blob/87e57b5390cb7cab569149ddb1eebfa04620c567/project/Dependencies.scala#L17


/cc @eed3si9n

**Comment by thesamet** [comment]
The fix can be tried in 0.10.4+28-3edb1755-SNAPSHOT
Note that the following line needs to be added to both `project/plugins.sbt` and `build.sbt`:

```
ThisBuild / resolvers += Resolver.sonatypeRepo("snapshots")
```
